spec/spec_helper.rb

```diff
@@ -134,7 +134,7 @@
 VCR.configure { |config| config.ignore_hosts(*driver_hosts) }
 
 VCR.configure do |c|
-  c.allow_http_connections_when_no_cassette = false
+  c.allow_http_connections_when_no_cassette = true
```

Comment:
Okay! So, this means that the failures before were just from the Pageviews fetching code eating the errors that VCR threw. We don't want to keep this setting enabled, though. Using mocks might still be a good strategy for the tests, or perhaps we can work around this in the test environment by making the pageview fetching behavior dependent on a setting, so that it skips doing it by default when updating ArticlesCourses records.